Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM manageiq/ruby:latest
+
+# For pg gem required by manageiq-gems-pending
+RUN yum -y install postgresql-devel
+
+RUN gem install bundler
+
+COPY docker-assets/* /opt/ssa_container/
+WORKDIR /opt/ssa_container
+
+# Should be moved into /opt/ssa_container
+RUN mkdir -p /opt/miq/log
```

Comment:
Let's change the path to `/opt/manageiq/amazon-smartstate`  (This mirrors how we do it for all of the other things [in our appliances](https://github.com/ManageIQ/manageiq-appliance-build/blob/0882883a7332247d3d238ec88d31564c80e43a54/kickstarts/partials/post/source_setup.ks.erb#L5))

Also, let's expose this path as an ENV var so that @hsong-rh can use that path from the code without having to worry about productization.  Alternately, it can be exposed to him via settings.